main.py

```
@@ -1,4 +1,4 @@
-from datetime import date
+from datetime import date, datetime
```

Ruff (0.12.2)

`datetime.date` imported but unused

Remove unused import: `datetime.date`

(F401)